app/src/main/java/org/mozilla/fenix/onboarding/JumpBackInCFRDialog.kt

```diff
@@ -50,12 +50,15 @@ class JumpBackInCFRDialog(val recyclerView: RecyclerView) {
 
     private fun createJumpCRF(anchor: View): Dialog? {
         val context: Context = recyclerView.context
-        if (!context.settings().showSyncCFR) {
+
+        if (context.settings().showSyncCFR) {
```

Comment:
I may have gotten this wrong, but assuming the `showSyncCFR` setting is enabled and no recent synced tab has been found on the first homescreen interaction, wouldn't this make the jump back in CFR not be displayed too (I believe this is the only case when we want to show the jump back in CFR)?

Reply:
Seems like the code gets a little tangled with all these settings for other CFRs that one CFR should't know or care about. If another CFR is added things might get unmaintainable with complex checks in every CFR & easy to miss details.
I'd like to bring into discussion the cfr manager approach that would be the only one knowing about and controlling all CFRs. It can be the one central point that knows when one is actually displayed to not show others and it can easily control the order in which they are shown.
Can also help with this.

Reply:
And now yet another CFR is sneaking in! https://github.com/mozilla-mobile/fenix/pull/26758/files#diff-f6968644d19314ed1385ed1bef14d8a3aaa163999143e11b0449b94292e6be56

this is getting officially confusing to read at a glance.

Reply:
Discussed with @Mugurell to followup with adding a manager to control all the CFRs that shows up. However, I don't believe we have a bug here since `createJumpCRF` is only called after it finds a view to attach to.